rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section11/rule_11_7.json

```diff
@@ -293,6 +293,296 @@
+                            }
+                        ],
+                        "type": "PROPOSED"
+                    }
+                ]
+            },
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_descriptions": [
+                    {
+                        "calendar": {
+                            "is_leap_year": false
+                        },
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "floor_name": "Floor1",
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "service_water_heating_area_type": "FIRE_STATION",
+                                                        "service_water_heating_uses": [
+                                                            {
+                                                                "id": "SHW 1",
+                                                                "served_by_distribution_system": "SHW Distribution 1",
+                                                                "use": 10,
+                                                                "use_units": "POWER_PER_AREA"
+                                                            }
+                                                        ]
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "service_water_heating_distribution_systems": [
+                            {
+                                "id": "SHW Distribution 1",
+                                "tanks": [
+                                    {
+                                        "id": "SHW Dist Tank 1"
+                                    }
+                                ]
+                            }
+                        ],
+                        "service_water_heating_equipment": [
+                            {
+                                "id": "SHW Equipment 1",
+                                "distribution_system": "SHW Distribution 1",
+                                "heater_fuel_type": "PROPANE",
```

Comment:
Can we update this value to ```NATURAL_GAS```? ```FIRE_STATION``` (line 533) is supposed to have the ```GAS_STORAGE_WATER_HEATER```.

Reply:
Hi @yunjoonjung-PNNL for rule d) the intent is to test this bit of code:

https://github.com/pnnl/ruleset-checking-tool/blob/7e8951f51ac40d1916bc8db4e00c32d322788a78/docs/section11/Rule11-7.md?plain=1#L41

Let me know if i'm misunderstanding the intent of the rule. As far as I can tell, this is saying to treat propane the same as gas. Is that incorrect?

Reply:
Juan - I just realized that I implemented the logic wrong. Thank you for finding this out. I'll merge this PR to my branch.